core/src/main/java/org/apache/accumulo/core/client/Connector.java

```diff
@@ -285,7 +283,7 @@ public abstract ConditionalWriter createConditionalWriter(String tableName,
    * Accessor method for internal instance object.
    *
    * @return the internal instance object
-   * @deprecated since 2.0.0, use {@link #info()} instead
+   * @deprecated since 2.0.0, use {@link AccumuloClient#info()} instead
```

Comment:
Not sure if this is needed since the entire class is deprecated now.  However it is helpful info.

Reply:
Yeah I was thinking having the link is helpful.